src/core/Tooltip/index.stories.tsx

```diff
@@ -75,6 +75,8 @@ Dark.parameters = {
 
 Dark.args = {
   arrow: true,
+  // delay allows for font to load and prevents chromatic from constantly creating new baselines
```

Comment:
Is there a way we can set this as a config option in chromatic, rather than setting it for several components?

Reply:
Thanks for the review @mmmmmmaya 
Yes we can, I'll update the PR 🤓